### DISCUSSION TITLE
Convert jpg to heic crash app

### DISCUSSION THREAD

**Discussion by pol2095** [dlemstra/Magick.NET#809] [Help]
### Prerequisites

- [x] I have written a descriptive issue title
- [X] I have verified that I am using the latest version of Magick.NET
- [X] I have searched [open](https://github.com/dlemstra/Magick.NET/issues) and [closed](https://github.com/dlemstra/Magick.NET/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
I try to convert jpg byte array to heic byte array, but my app crash

### Steps to Reproduce
```
public byte[] jpgToHeic(byte[] jpg)
{
	var ms = new MemoryStream(jpg);
	using (MagickImage i = new MagickImage(ms))
	{
		i.Format = MagickFormat.Heic;
		
		using (MemoryStream memStream = new MemoryStream(i.ToByteArray()))
		{
			return memStream.ToArray();
		}
	}
}
```
the same code using "MagickFormat.Png" work correctly

### System Configuration
Visual Studio 2019

- Magick.NET-Q8-AnyCPU version: v7.22.2.2
- Windows 10 last version

**Comment by dlemstra** [comment]
You are getting the following exception:

```
ImageMagick.MagickMissingDelegateErrorException : no encode delegate for this image format `HEIC' @ error/constitute.c/WriteImage/1252
```

This is happening because there is no support for writing heic images. We do recently support AVIF though.

**Reply by pol2095** [reply]
is-it planned to support it ?
I can use AVIF, thanks.

**Reply by dlemstra** [reply]
There are not plans for adding support for this at the moment.
